lib/WorseReflection/Core/Inference/Resolver/StringLiteralResolver.php

```diff
@@ -30,4 +31,23 @@ public function resolve(NodeContextResolver $resolver, Frame $frame, Node $node)
             ]
         );
     }
+
+    private function getStringContentsText(StringLiteral $node): string
+    {
+        $children = $node->children;
+        if ($children instanceof Token) {
+            $value = (string)$children->getText($node->getFileContents());
+            $startQuote = substr($node, 0, 1);
+
+            if ($startQuote === '\'') {
+                return rtrim(substr($value, 1), '\'');
+            }
+
+            if ($startQuote === '"') {
+                return rtrim(substr($value, 1), '"');
+            }
+        }
+
+        return '';
```

Comment:
the TolerantParser method to get strings includes the quotes... but probably escapes text? maybe combine the two.